src/mutableAtom/index.ts

```diff
@@ -0,0 +1,155 @@
+import { Getter, Setter, atom } from 'jotai'
+import assert from 'minimalistic-assert'
+import { proxy, snapshot, subscribe } from 'valtio'
+import {
+  Options,
+  PromiseOrValue,
+  ProxyRef,
+  ProxyState,
+  SetCb,
+  SetSelf,
+  Store,
+  Wrapped,
+  WriteFn,
+} from './types'
+
+export function mutableAtom<Value>(
+  value: Value,
+  options: Options<Value> = defaultOptions
+) {
+  const { proxyFn } = { ...defaultOptions, ...options }
+
+  const valueAtom = atom({ value })
+
+  const storeAtom = atom(
+    () => ({ unsubscribe: null, isMounted: false } as Store),
+    (get, set, writeFn: WriteFn) => writeFn(get, set)
```

Comment:
It would be easier for me to read to explicitly get an action, rather than leaking get and set outside:
```js
    (get, set, action: 'mount' | 'unmount') => {
      // ...
    }
```